RELEASES.md

```diff
@@ -209,7 +209,7 @@ containerd versions:
 | GRPC API         | Stable   | 1.0                | [gRPC API](#grpc-api) |
 | Metrics API      | Stable   | 1.0                | - |
 | Runtime Shim API | Stable   | 1.2                | - |
-| Daemon Config    | Stable   | 1.0                | - |
+| Daemon Config v2 | Stable   | 1.5                | [man page](https://github.com/containerd/containerd/blob/main/docs/man/containerd-config.toml.5.md) |
```

Comment:
Why 1.5?

Reply:
I was helping @sbuckfelder with this; the table layout seems to imply that you put the containerd version in which that feature "stabilized"; looks like `version = 2` was announced in the 1.5.0 release notes, so I had him put 1.5 to match what I think the rest of the rows are trying to show.

Reply:
or 1.5 (_config v2_)
using the pattern below in line 213..

Reply:
Didn't we introduce config v2 in containerd v1.3?
https://github.com/containerd/containerd/pull/3322

Not sure when it was considered stable at that time though.

Reply:
wasn't sure either .. in CRI we migrated to the v2 config during development of the containerd v1.3 release, can't say if all the components have moved up or not.. and v1 backwards compatibility is still there ...  Not sure how much detail is needed.

Reply:
@dmcgowan related to the table versioning changes